src/index.js

```diff
@@ -43,6 +44,24 @@ $('#js-solve-button').on('click', function() {
 
 $('#js-submit-button').on('click', function(e) {
   e.preventDefault();
-  domUpdates.checkUserGuess(game);
+  let letterInput = domUpdates.grabCurrentLetter();
+  game.currentPuzzle.checkUserGuess(letterInput);
+  $('.input').val('');
+});
+
+$('#js-vowel-submit').on('keyup', function(e) {
+  e.preventDefault();
+  domUpdates.checkVowel();
```

Comment:
Not sure if you guys are currently using this function?
If you plan on using it, that's fine.